schemachange/sc_add_table.c

```diff
@@ -249,9 +230,7 @@ int do_add_table(struct ireq *iq, struct schema_change_type *s,
         return rc;
     }
 
-    if (!(db = get_dbtable_by_name(s->table))) return SC_INTERNAL_ERROR;
```

Comment:
I think this was only a check that we can find dbtable by name at this point.

Reply:
Yes, but I made changes so that we dont add db to thedb->dbs until we finalize. So with my changes, this function will fail.